### PR TITLE
deprecate expandAsync and expandSync for the renamed corresponding me…

### PR DESCRIPTION
**Pull request by alabiaga**
- deprecate expandSync and expandAsync methods
- replace deprecated calls with renamed expandUrlSync and expandUrlAsync

Fixes/Closes/Related-to #6322

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,7 +79,7 @@
     "formidable": "1.1.1",
     "fs-extra": "5.0.0",
     "glob": "7.1.2",
-    "gulp": "3.9.1",
+    "gulp": "^3.9.1",
```
Comment (choumx) [inline package.json]: Why this change?

Comment (alabiaga) [reply package.json]: oh this was not intentional but the current state of this file has this change as well. I am guessing it was from running yarn.

Comment (alabiaga) [reply package.json]: actually im wrong. reverting file.
